adviser/base/argo-workflows/trigger-integration.yaml

```diff
@@ -14,9 +14,9 @@ spec:
           - name: "THOTH_ADVISER_METADATA"
       outputs:
         parameters:
-          - name: origin
+          - name: metadata_origin
```

Comment:
if we are changing this value, we should change this as well  https://github.com/thoth-station/thoth-application/blob/c45e9f6395d576de8e0e346f29c1a66959a7e968/adviser/base/openshift-templates/adviser.yaml#L281
right ?

Reply:
nice catch! thanks!!